api/libproject/devstart.rb

```diff
@@ -2402,8 +2402,10 @@ def deploy(cmd_name, args)
   common = Common.new
   common.status "Running database migrations..."
   ENV.update(read_db_vars(gcc))
-  migrate_database(op.opts.dry_run)
-  load_config(ctx.project, op.opts.dry_run)
```

Comment:
looks like 'ctx' has only one reference left in this file, and I don't see that it's defined anywhere.  Let's clean this up in another PR.

Reply:
oh... yes I misread that.  Thanks.